package.json

```diff
@@ -18,7 +18,7 @@
         "dotenv": "^16.3.1",
         "googleapis": "^126.0.1",
         "mongodb": "^5.8.1",
-        "npx": "^10.2.2",
+        "npx": "^3.0.0",
```

Comment:
Why would we use a npx version from 2016?

Reply:
I have no idea why it decided to downgrade the version, I must have typed the wrong npm command